apps/admin-gui/src/app/vos/pages/member-detail-page/member-overview/member-overview.component.html

```diff
@@ -68,17 +68,29 @@ <h1 class="page-subtitle">{{'MEMBER_DETAIL.OVERVIEW.RESET_PASSWORD' | translate}
           </div>
         </mat-card-content>
       </mat-card>
-      <div *ngIf="member.sponsored">
-        <mat-card appearance="outlined" class="mat-elevation-z3 mt-4">
-          <mat-card-header>
-            <mat-card-title>
-              <h1 class="page-subtitle">{{'MEMBER_DETAIL.OVERVIEW.SPONSORSHIP' | translate}}</h1>
-            </mat-card-title>
-            <mat-card-subtitle class="mt-n3">
-              {{'MEMBER_DETAIL.OVERVIEW.SPONSORED_BY' | translate}}
-            </mat-card-subtitle>
-          </mat-card-header>
-          <mat-card-content *ngIf="sponsors.length !== 0">
+
+      <mat-card
+        appearance="outlined"
+        class="mat-elevation-z3 mt-4"
+        *ngIf="member.sponsored || enableSponsorButton()">
+        <mat-card-header>
+          <mat-card-title>
+            <h1 class="page-subtitle">{{'MEMBER_DETAIL.OVERVIEW.SPONSORSHIP' | translate}}</h1>
+          </mat-card-title>
+          <mat-card-subtitle class="mt-n3 mb-2" *ngIf="member.sponsored">
+            {{'MEMBER_DETAIL.OVERVIEW.SPONSORED_BY' | translate}}
+          </mat-card-subtitle>
+        </mat-card-header>
+        <mat-card-content>
+          <button
+            *ngIf="enableSponsorButton()"
+            (click)="sponsorThisMember()"
+            class="action-button"
+            color="accent"
+            mat-flat-button>
+            {{'MEMBER_DETAIL.OVERVIEW.SPONSOR_MEMBER'|translate}}
+          </button>
+          <div *ngIf="member.sponsored" class="mt-3">
             <h6 class="d-flex">
               {{'MEMBER_DETAIL.OVERVIEW.SPONSORS' | translate}}:
```

Comment:
Please consider aligning the header with the edit icon as part of this task, it looks bad
![Screenshot from 2023-07-10 12-36-05](https://github.com/CESNET/perun-web-apps/assets/97669007/14a57bd6-c237-4ecc-95e0-bfdbaae29ca9)
Also 'This member is sponsored' followed by a 'Sponsor this member' button looks a bit weird. I feel like this sentence is unnecessary now. What do you think @bodnara  @sarkapalkovicova ?

Reply:
Agree, showing current sponsors should be sufficient, or?

Reply:
The design for this use case does include these texts. However it does not seem to add that much value.
![image](https://github.com/CESNET/perun-web-apps/assets/23495969/915a2611-8d47-4c14-ada6-f36409cf207c)
![image](https://github.com/CESNET/perun-web-apps/assets/23495969/24ed3e0b-ced5-4785-a829-c9dc8f46b7b1)